### DISCUSSION TITLE
Drawing quantum circuit after solving system.

### DISCUSSION THREAD

**Discussion by HuynhTran0301** [QuantumSavory/Quantikz.jl#13] [Q&A]
I would like to ask that may Quantikz.jl help to draw the quantum circuit after solving the ODE system?

**Comment by Krastanov** [comment]
If you can specify the sequence of gates, Quantikz.jl can draw them. But this is just a visualization package. It has nothing to do with ODEs.

If you are solving some Hamiltonian evolution of a set of qubits (what I assume you mean by an ODE system), maybe you will get some unitary operation out of it. There is no circuit involved here.

Each unitary has multiple circuits that can implement it, but finding one of these circuits is not straightforward, especially if you have a large number of qubits.

**Reply by HuynhTran0301** [reply]
I mean, I have used solver such as QuLDE() to solve the ODE problem. May Quantikz help draw the system's quantum circuit after solving it?. Because the solver automatically simulates quantum circuits and solves it.
